### PR TITLE
Update task unit test

### PR DESCRIPTION
**Pull request by Asuraxsoul**

### CODE REVIEW

File: src/main/java/seedu/jarvis/model/task/Event.java [hunk 1]
```diff
@@ -46,6 +46,8 @@ public Event(String taskId, String description, String dateTime) {
         this.dateTime = LocalDateTime.parse(replacedDateTime, dateTimeFormat);
     }
 
+    // ------------------- Getters -----------------------------------------------------------------------------
```
Comment (ngzhenteng) [inline src/main/java/seedu/jarvis/model/task/Event.java:49]: Thanks for marking out the code to make it more readable.

File: src/test/java/seedu/jarvis/model/task/DeadlineTest.java [hunk 2]
```diff
@@ -41,6 +41,13 @@ void getDateTime() {
         assertFalse(deadlineTestTwo.getDateTime().equals(FORMATTED_DATETIME_THREE));
     }
 
+    @Test
+    void generateTaskIdTest() {
```
Comment (ngzhenteng) [inline src/test/java/seedu/jarvis/model/task/DeadlineTest.java]: Great addition of test cases! Perhaps this could be generateTaskId_success()

**Review by ngzhenteng — COMMENTED** [review]
LGTM thanks for adding these tests!

File: src/test/java/seedu/jarvis/model/task/EventTest.java [hunk 3]
```diff
@@ -41,6 +41,13 @@ void getDateTime() {
         assertFalse(eventTestFour.getDateTime().equals(FORMATTED_DATETIME_ONE));
     }
 
+    @Test
+    void generateTaskIdTest() {
```
Comment (ngzhenteng) [inline src/test/java/seedu/jarvis/model/task/EventTest.java]: Perhaps a name like: generateTaskId_nextId_notEqualPrevious()

File: src/test/java/seedu/jarvis/model/task/DeadlineTest.java (hunk 2 above)
Comment (Asuraxsoul) [reply src/test/java/seedu/jarvis/model/task/DeadlineTest.java]: okay noted

**Review by ngzhenteng — COMMENTED** [review]
LGTM!

File: src/test/java/seedu/jarvis/model/task/EventTest.java (hunk 3 above)
Comment (Asuraxsoul) [reply src/test/java/seedu/jarvis/model/task/EventTest.java]: noted! Thanks

**Pull request merged**